pkg/api/dump.go

```diff
@@ -254,11 +313,27 @@ func (s *service) containerdDump(ctx context.Context, imagePath, containerID str
 	return s.postDump(ctx, imagePath, state, nil)
 }
 
-func (s *service) setupStreamerCapture(dumpdir string) (*exec.Cmd, error) {
+func (s *service) setupStreamerCapture(dumpdir string, num_pipes int32) (*exec.Cmd, error) {
```

Comment:
should output cmd.Stderr and cmd.Stdout to our logger so all cedana logs (including the ones from the streamer) are accessible from one place